backend/globaleaks/jobs/notification_sched.py

```diff
@@ -120,9 +120,15 @@ def process_ReceiverFile(self, store, rfile, data):
         self.process_mail_creation(store, data)
 
     def process_mail_creation(self, store, data):
+        receiver_id = data['receiver']['id']
+
+        # Do not spool emails if the receiver has opted out of ntfns for this tip.
+        if not data['tip']['enable_notifications']:
+          log.debug("Discarding emails for %s due to receiver's preference." % receiver_id)
```

Comment:
'receiver' is generally nowadays used only in the code; if the message is for the user(of the webapplication or the admin that looks the logs) the texts should prefer using 'recipient'. various translators  indipendently suggested this.